less/mixins.less

```diff
@@ -1 +1,27 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.button(@width, @fontcolor) {
+    background-color: @fontcolor;
+    color: #fff;
+    width: @width;
+    font-size: 1.6rem;
+    border-radius: 5px;
+    text-align: center;
+    padding: 15px 2%;
+    transition: ease-in-out 0.250s;
+    cursor: pointer;
+    &:hover {
+        background-color: #fff;
+        color: @fontcolor;
+    }
+}
+
+.flex(@justify) {
+    display: flex;
+    justify-content: @justify;
+}
+
+.flexwrap() {
+    display: flex;
+    flex-wrap: wrap;
+}
```

Comment:
You really nailed these mixins. Great job! The great thing about mixins, you can use these over and over again in future projects. No need to reinvent the wheel every time. You can grab this mixin file and drop it into any LESS project, and boom. You just added button and flex in two seconds, and it's your own code instead of something you found on the internet. Very cool!